### ISSUE TITLE
Sequenced keys problem

### ISSUE THREAD

**Issue by JeroenSack** [synergy/synergy#4625]
When writing sequenced keys that have to be typed with shift (" and ~) you will get the lower character when not holding shift when pressing the second key. So for example I press shift + ", then release shift and type space, I will get ' on the screen. However when I press shift + " and hold shift while pressing space, I will get " on the screen.

The server I am using is Windows 7 64bit with Synergy 1.7.2 and client is Mac OS X 10.10 with Synergy 1.7.2. This problem also occured with Synergy 1.7.1.

**Comment by speaker** [comment]
I don't think I understand this fully. Is this in regards to windows multi-lingual sequenced keys setups?

**Comment by JeroenSack** [comment]
Yes, that is what I mean, for when you want to type an é for example. This doesn't work on the mac, but using windows as a server will still have it functioning.

**Comment by speaker** [comment]
@JeroenSack , Got it, thank you for the information. This is a really tough problem to crack because of the number of possible combinations that exist between languages, keyboard mappings, and OSs. We even sometimes see problem when an OS is installed in Klingon, for example, but then changed to Tengwar, the system might still report as Klingon. 

We're looking into ways to try and address this issue better.
